### PR TITLE
fix some dark theme bugs

### PR DESCRIPTION
**Pull request by xaylpq**

### CODE REVIEW

File: resources/styles/applyStyles.css [hunk 1]
```diff
@@ -1,9 +1,14 @@
 /*  styles.css  */
+:root {
```
Comment (stasguryev) [inline resources/styles/applyStyles.css]: variables.css?

File: resources/styles/theme.css [hunk 2]
```diff
@@ -532,18 +538,18 @@ svg.svg-grid-icons {
 .checkbox:not(:empty):hover {
   background-color: var(--theme-colors__primary_fade);
   border-color: var(--theme-colors__primary_fade);
-  color: var(--theme-colors__primary-txt);
+  color: var(--theme-colors__btn-txt);
```
Comment (stasguryev) [inline resources/styles/theme.css]: ?

File: resources/styles/theme.css [hunk 3]
```diff
@@ -532,18 +538,18 @@ svg.svg-grid-icons {
...
 }
 
 .grid-header .checkbox:not(:empty) {
-  color: var(--theme-colors__primary);
-  border-color: var(--theme-colors__main_fadest);
-  background-color: var(--theme-colors__main_fadest);
+  color: var(--theme-colors__primary-txt);
+  border-color: var(--border-grids-color);
```
Comment (stasguryev) [inline resources/styles/theme.css]: почему?

File: src/components/toolbar/views/blinkCheckbox/BlinkCheckboxPopoutPanelView.js [hunk 4]
```diff
@@ -43,11 +43,11 @@ export default Marionette.CollectionView.extend({
     template: Handlebars.compile(template),
 
     templateContext() {
-        const isSomeHidden = this.collection.some(model => model.get('isHidden'));
+        const isEveryHidden = this.collection.every(model => model.get('isHidden'));
```
Comment (stasguryev) [inline src/components/toolbar/views/blinkCheckbox/BlinkCheckboxPopoutPanelView.js]: ?

File: src/components/toolbar/views/blinkCheckbox/BlinkCheckboxVisibilityView.js [hunk 5]
```diff
@@ -62,13 +62,13 @@ export default Marionette.View.extend({
     },
 
     __displayVisibility(isVisible) {
-        const filterIcon = this.ui.filterIcon;
+        const filterIcon = this.el.classList;
```
Comment (stasguryev) [inline src/components/toolbar/views/blinkCheckbox/BlinkCheckboxVisibilityView.js]: filterIcon => elClassList

File: src/form/editors/MembersSplitEditorView.ts [hunk 6]
```diff
@@ -116,7 +116,7 @@ export default (formRepository.editors.MembersSplit = BaseEditorView.extend({
             header: this.getOption('title') || Localizer.get('CORE.FORM.EDITORS.MEMBERSPLIT.MEMBERSTITLE'),
             buttons: [
                 {
-                    id: false,
```
Comment (stasguryev) [inline src/form/editors/MembersSplitEditorView.ts:119]: why remove?

File: src/list/views/GridView.ts [hunk 7]
```diff
@@ -732,7 +732,7 @@ export default Marionette.View.extend({
     },
 
     __confirmUserAction(text: string = '', title: string = '', yesButtonText: string = 'Yes', noButtonText: string = 'No') {
-        return Core.services.MessageService.showMessageDialog(text, title, [{ id: false, text: noButtonText }, { id: true, text: yesButtonText }]);
```
Comment (stasguryev) [inline src/list/views/GridView.ts:735]: id: false + isCancel: true

File: resources/styles/theme.css (hunk 2 above)
Comment (xaylpq) [reply resources/styles/theme.css]: Done

File: src/components/toolbar/views/blinkCheckbox/BlinkCheckboxPopoutPanelView.js (hunk 4 above)
Comment (xaylpq) [reply src/components/toolbar/views/blinkCheckbox/BlinkCheckboxPopoutPanelView.js]: Done

File: src/components/toolbar/views/blinkCheckbox/BlinkCheckboxVisibilityView.js (hunk 5 above)
Comment (xaylpq) [reply src/components/toolbar/views/blinkCheckbox/BlinkCheckboxVisibilityView.js]: Done

File: src/form/editors/MembersSplitEditorView.ts (hunk 6 above)
Comment (xaylpq) [reply src/form/editors/MembersSplitEditorView.ts:119]: Done

File: src/list/views/GridView.ts (hunk 7 above)
Comment (xaylpq) [reply src/list/views/GridView.ts:735]: Done

File: resources/styles/applyStyles.css (hunk 1 above)
Comment (xaylpq) [reply resources/styles/applyStyles.css]: Done

**Pull request merged**